README.md

```diff
@@ -62,10 +62,11 @@ for that dataset. The following is an example of a dataset object:
 
 ```json
 {
+  "eventsTable": "indonesia_v20181029",
   "pipeline": "pipe_indonesia_production_v20181029",
-  "bigquery": {"
-    fishingTable": "world-fishing-827.pipe_indonesia_production_v20181029.published_events_fishing"
-  }
+  "tracksTable": "indonesia_v20181029_tracks",
+  "vesselIndex": "andres"-test-indonesia,
+  "vesselsTable": "indonesia_v20181029_vessels"
```

Comment:
@andres-arana what's the `vesselsTable` used for?

Reply:
I'll add those docs to a wiki page.